app/code/Magento/Checkout/Model/Session.php

```diff
@@ -568,4 +569,22 @@ protected function isQuoteMasked()
     {
         return $this->isQuoteMasked;
     }
+
+    /**
+     * @return CartInterface|false
+     */
+    private function getQuoteByCustomer()
```

Comment:
I think it's better to return null in case when quote wasn't found. It will allow us to define return type 
```suggestion
    private function getQuoteByCustomer(): ?CartInterface
```

But we need to change comparison above.